osu.Game/OsuGame.cs

```diff
@@ -182,7 +182,26 @@ private void load(FrameworkConfigManager frameworkConfig)
             // bind config int to database SkinInfo
             configSkin = LocalConfig.GetBindable<int>(OsuSetting.Skin);
             SkinManager.CurrentSkinInfo.ValueChanged += skin => configSkin.Value = skin.NewValue.ID;
-            configSkin.ValueChanged += skinId => SkinManager.CurrentSkinInfo.Value = SkinManager.Query(s => s.ID == skinId.NewValue) ?? SkinInfo.Default;
+            configSkin.ValueChanged += skinId =>
```

Comment:
```csharp
            configSkin.BindValueChanged(skinId => ..., true);
```

Reply:
?

Reply:
instead of
```csharp
configSkin.ValueChanged += ...;
configSkin.TriggerChange();
```

Reply:
kind of out of scope of this PR, but sure.